packages/experimental/ActivityIndicator/src/ActivityIndicator.tsx

```diff
@@ -75,15 +79,19 @@ export const ActivityIndicator = (props: ActivityIndicatorProps) => {
     outputRange: ['0deg', '359deg'],
   });
 
-  // or display: none, but since React Native ActivityIndicator still takes up space when hidden, using opacity
-  let hideOpacity = 1;
+  // React Native ActivityIndicator still takes up space when hidden, so to perfectly match would use opacity
+  // using display: 'none' because screen reader might still read it when hidden with opacity
+  let displayValue = 'flex';
```

Comment:
```const displayValue = (props.animating == false && props.hidesWhenStopped == true) ? 'flex': 'none'```

We prefer consts and immutability when possible to eliminate the class of bugs where a variable can be reassigned to a new value after it has been created. It's also perfectly valid to have conditional initialization like I did.